reactnative/reactnative-todo/.env.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Avoid trailing slashes in the URL string values below
+ */
+const API_PORT = 9443;
+const API_BASE_URL = 'http://10.0.2.2'; // for iOS - http://localhost
```

Comment:
Do we need this ip address and port here? Can we replace with placeholder?

Reply:
fixed as a placeholder.